packages/astroport-governance/src/utils.rs

```diff
@@ -7,8 +7,12 @@ pub const MAX_LOCK_TIME: u64 = 2 * 365 * 86400; // 2 years (104 weeks)
 /// The constant describes the maximum number of accounts for claim.
 pub const CLAIM_LIMIT: u64 = 10;
 
+const PERIOD_SHIFT: u64 = 3 * 86400;
```

Comment:
A more common approach is setting start of the epoch, instead of adding 3 days to beggining of timestamp.
Maybe let's just use some date: e.g (1643587200 - Monday, 31 January 2022 г., 0:00:00) and change the formula to 

(time - EPOCH_START) / WEEK

Reply:
Ok, fixed